<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="4" passed="1" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[====db conncetion sucessful====]]>
    </line>
    <line>
      <![CDATA[====Browser launch sucessful=====]]>
    </line>
    <line>
      <![CDATA[====Login sucessful=====]]>
    </line>
    <line>
      <![CDATA[====Browser launch sucessful=====]]>
    </line>
    <line>
      <![CDATA[====Login sucessful=====]]>
    </line>
    <line>
      <![CDATA[createContactWithLeadSourceTest------TestScript failed]]>
    </line>
    <line>
      <![CDATA[====Logout sucessful=====]]>
    </line>
    <line>
      <![CDATA[====Browser close sucessful=====]]>
    </line>
    <line>
      <![CDATA[====Database close sucessful=====]]>
    </line>
  </reporter-output>
  <suite started-at="2022-03-16T17:13:45 IST" name="Suite" finished-at="2022-03-16T17:18:32 IST" duration-ms="287403">
    <groups>
    </groups>
    <test started-at="2022-03-16T17:13:45 IST" name="Test" finished-at="2022-03-16T17:18:32 IST" duration-ms="287403">
      <class name="com.crm.ContactTestcaseTest.CreateContactWithoutMandatoryTest">
        <test-method is-config="true" signature="connectDataBase()[pri:0, instance:com.crm.ContactTestcaseTest.CreateContactWithoutMandatoryTest@6283d8b8]" started-at="2022-03-16T17:13:45 IST" name="connectDataBase" finished-at="2022-03-16T17:13:45 IST" duration-ms="20" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[====db conncetion sucessful====]]>
            </line>
          </reporter-output>
        </test-method> <!-- connectDataBase -->
        <test-method is-config="true" signature="launchTheBrowser()[pri:0, instance:com.crm.ContactTestcaseTest.CreateContactWithoutMandatoryTest@6283d8b8]" started-at="2022-03-16T17:13:50 IST" name="launchTheBrowser" finished-at="2022-03-16T17:15:11 IST" duration-ms="80260" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[====Browser launch sucessful=====]]>
            </line>
          </reporter-output>
        </test-method> <!-- launchTheBrowser -->
        <test-method is-config="true" signature="login()[pri:0, instance:com.crm.ContactTestcaseTest.CreateContactWithoutMandatoryTest@6283d8b8]" started-at="2022-03-16T17:15:11 IST" name="login" finished-at="2022-03-16T17:16:13 IST" duration-ms="62231" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[====Login sucessful=====]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="createContactWithoutMandatoryTest()[pri:0, instance:com.crm.ContactTestcaseTest.CreateContactWithoutMandatoryTest@6283d8b8]" started-at="2022-03-16T17:16:13 IST" name="createContactWithoutMandatoryTest" finished-at="2022-03-16T17:17:11 IST" duration-ms="58303" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createContactWithoutMandatoryTest -->
        <test-method is-config="true" signature="logout()[pri:0, instance:com.crm.ContactTestcaseTest.CreateContactWithoutMandatoryTest@6283d8b8]" started-at="2022-03-16T17:17:11 IST" name="logout" finished-at="2022-03-16T17:17:11 IST" duration-ms="78" status="FAIL">
          <exception class="org.openqa.selenium.NoSuchSessionException">
            <message>
              <![CDATA[Session ID is null. Using WebDriver after calling quit()?
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-FUABKC3', ip: '192.168.43.254', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.1'
Driver info: driver.version: RemoteWebDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchSessionException: Session ID is null. Using WebDriver after calling quit()?
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'DESKTOP-FUABKC3', ip: '192.168.43.254', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.1'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:125)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.perform(RemoteWebDriver.java:618)
at org.openqa.selenium.interactions.Actions$BuiltAction.perform(Actions.java:639)
at org.openqa.selenium.interactions.Actions.perform(Actions.java:595)
at com.crm.GenericLibrary.WebDriverUtility.mouseHover(WebDriverUtility.java:108)
at com.crm.ObjectRepository.HomePage.signOutOfApp(HomePage.java:98)
at com.crm.GenericLibrary.BaseClass.logout(BaseClass.java:92)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:676)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:660)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchSessionException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.crm.ContactTestcaseTest.CreateContactWithoutMandatoryTest@6283d8b8]" started-at="2022-03-16T17:17:11 IST" name="closeBrowser" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1647431231841" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="closeDb()[pri:0, instance:com.crm.ContactTestcaseTest.CreateContactWithoutMandatoryTest@6283d8b8]" started-at="2022-03-16T17:18:36 IST" name="closeDb" finished-at="2022-03-16T17:18:36 IST" duration-ms="1" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[====Database close sucessful=====]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeDb -->
      </class> <!-- com.crm.ContactTestcaseTest.CreateContactWithoutMandatoryTest -->
      <class name="com.crm.ContactsTest.CreateContactWithLeadSourceTest">
        <test-method is-config="true" signature="launchTheBrowser()[pri:0, instance:com.crm.ContactsTest.CreateContactWithLeadSourceTest@5acf93bb]" started-at="2022-03-16T17:17:11 IST" name="launchTheBrowser" finished-at="2022-03-16T17:17:42 IST" duration-ms="30998" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[====Browser launch sucessful=====]]>
            </line>
          </reporter-output>
        </test-method> <!-- launchTheBrowser -->
        <test-method is-config="true" signature="login()[pri:0, instance:com.crm.ContactsTest.CreateContactWithLeadSourceTest@5acf93bb]" started-at="2022-03-16T17:17:42 IST" name="login" finished-at="2022-03-16T17:18:01 IST" duration-ms="18233" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[====Login sucessful=====]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="createContactWithLeadSourceTest()[pri:0, instance:com.crm.ContactsTest.CreateContactWithLeadSourceTest@5acf93bb]" started-at="2022-03-16T17:18:01 IST" name="createContactWithLeadSourceTest" finished-at="2022-03-16T17:18:26 IST" duration-ms="25666" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.fail(Assert.java:106)
at com.crm.ContactsTest.CreateContactWithLeadSourceTest.createContactWithLeadSourceTest(CreateContactWithLeadSourceTest.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[createContactWithLeadSourceTest------TestScript failed]]>
            </line>
          </reporter-output>
        </test-method> <!-- createContactWithLeadSourceTest -->
        <test-method is-config="true" signature="logout()[pri:0, instance:com.crm.ContactsTest.CreateContactWithLeadSourceTest@5acf93bb]" started-at="2022-03-16T17:18:27 IST" name="logout" finished-at="2022-03-16T17:18:31 IST" duration-ms="4310" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[====Logout sucessful=====]]>
            </line>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:com.crm.ContactsTest.CreateContactWithLeadSourceTest@5acf93bb]" started-at="2022-03-16T17:18:31 IST" name="closeBrowser" finished-at="2022-03-16T17:18:32 IST" duration-ms="1136" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[====Browser close sucessful=====]]>
            </line>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.crm.ContactsTest.CreateContactWithLeadSourceTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
